gigareview/common/models/EMReportJob.php

```diff
@@ -14,9 +14,82 @@ class EMReportJob extends \yii\base\BaseObject implements \yii\queue\JobInterfac
     public string $fetchDate ;
     public string $scope;
 
+
     public function execute($queue)
     {
-        // TODO: Implement execute() method.
+        if ($this->scope === "manuscripts") {
+            $this->executeManuscriptJob($this->content);
+        }
+            //TODO: Will be implemented in ticket no. #1065
+//            $ingest = Ingest::findOne(["report_type" => "1", "parse_status" => null]);
+//            $ingest->remote_file_status = Ingest::REMOTE_FILES_STATUS_EXISTS;
+//            $ingest->parse_status = Ingest::PARSE_STATUS_YES;
+//            $ingest->update();
+
+//        }
+            //TODO: Will be implemented in ticket no. #1065
+//        elseif ($this->scope === "manuscripts" && $this->content === "No Results") {
+//            $ingest = Ingest::findOne(["report_type" => "1", "parse_status" => null]);
+//            $ingest->remote_file_status = Ingest::REMOTE_FILES_STATUS_NO_RESULTS;
+//            $ingest->parse_status = Ingest::PARSE_STATUS_NO;
+//            $ingest->update();
+//        }
+    }
+
+    /**
+     * @param string $content
+     * @return void
+     */
```

Comment:
If we have DocBlock then a simple one line summary describing what the function does would be good in addition to listing the input and output arguments.